src/test/java/org/jvnet/hudson/test/RestartableJenkinsRuleTest.java

```diff
@@ -52,4 +74,35 @@ public void testPortReuse() throws Exception {
                     assertEquals(port.get(), portReuse.j.getURL().getPort());
                 });
     }
+
+    @Test
+    @WithPluginManager(UnitTestSupportingPluginManager.class)
```

Comment:
Maybe Javadoc of `RestartableJenkinsRule` should note this idiom?

Reply:
happy to do that as a follow-up if it means I can get a release sooner